src/pages/surveys/[id]/questions/[order].page.tsx

```diff
@@ -0,0 +1,79 @@
+import type { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import { useContext, useEffect, useState } from 'react';
+
+import Button from 'components/Button';
+import PageLoader from 'components/PageLoader';
+import SurveyQuestion from 'components/SurveyQuestion';
+import useSession from 'hooks/useSession';
+import { SurveyQuestion as SurveyQuestionInterface } from 'services/surveyInterfaces';
+import { StoreContext } from 'store';
+
+import styles from './[order].module.css';
+
+const Question: NextPage = () => {
+  useSession();
+  const router = useRouter();
+  const questionOrder = parseInt(router.query.order as string);
```

Comment:
```suggestion
  const currentQuestionOrder = parseInt(router.query.????? as string);
```
I'm definitely not fond of using the name `order` (which should hold some values like "ascending", "descending" or "alphabetical" ^^) when handling indexes or ranks... But I saw the API uses this vocabulary, so there is little I can say :D 

Though, I would still recommend prefixing this variable with "current", so that it makes it explicit that this is the **current** question order/index. This also makes it consistent with the variables "currentQuestion" and "currentSurvey" (as otherwise, we could name them question and survey respectively)

Reply:
I update the page name to `currentOrder`, my first thought is to update it to `questionOrder` but it's already scope under `question` directory. So I think it makes more sense to me to name it `currentOrder`

Updated in [b9663ef](https://github.com/hanam1ni/react-surveyor/pull/78/commits/b9663ef1d1b59b478c3c1935175a0001c035edb4)

Reply:
Good alternative! :)